web_src/js/features/repo-issue.js

```diff
@@ -287,6 +287,33 @@ export function initRepoPullRequestMergeInstruction() {
   });
 }
 
+export function initRepoPullRequestAllowEditsFromMaintainers() {
+  $('#allow-edits-from-maintainers input').on('change', function () {
+    const $label = $('#allow-edits-from-maintainers label');
+    let url = $label.data('url');
+    if (this.checked) { // allow edits
+      url = `${url}/allow_edits`;
+    } else { // disallow edits
+      url = `${url}/disallow_edits`;
+    }
+    $.ajax({
+      url,
+      type: 'POST',
+      data: {
+        _csrf: csrfToken
+      },
+      error: () => {
+        const $labelElem = document.getElementById('allow-edits-from-maintainers-label');
+        $label.popup('destroy');
+        const oldContent = $labelElem.getAttribute('data-content');
+        $labelElem.setAttribute('data-content', $label.data('failed'));
+        $label.popup('show');
+        $labelElem.setAttribute('data-content', oldContent || '');
```

Comment:
That's too complex for showing an error message 😂

Reply:
It's  copied from the clipboard js function which does it the sane way

Reply:
Yep, I knew the code came from there (I also maintained that code recently)

But that code doesn't fit this requirement in my mind. The `popup` document:

https://fomantic-ui.com/modules/popup.html#specifying-content-in-javascript

Reply:
@wxiaoguang Do you know a way to show such a pop up with a custom message? I tried it with these in the docs you linked, but it didn't work.

Reply:
Since you are using Fomantic UI (`ui checkbox`), you could just follow the Fomantic UI document. 

I proposed a demo for it, you can use https://github.com/wxiaoguang/gitea/commit/6ea2bdfae2726324bd02a065e8e9ad27634889bf as a PR